WebService1/src/main/kotlin/at/serviceengineering/webservice1/services/CarService.kt

```diff
@@ -45,7 +44,7 @@ class CarService(
             carRepository.save(car)
             accountRepository.save(account)
         } catch (e: Exception) {
-            account.rentals?.remove(request)
+            account.rentals?.remove(rental)
```

Comment:
@JoeNeu bitte checken ob das passt. Geändert weil ich nen da compiler Error hatte

`Kotlin: Type inference failed. The value of the type parameter T should be mentioned in input types (argument types, receiver type or expected type). Try to specify it explicitly.`

wsl. wegen meiner anderen Kotlin version, but idk..

Reply:
Ja absolut! Gar nicht aufgefallen, das wäre wack af